### PR TITLE
Make target CPU percentage configurable in values

### PR DESCRIPTION
**Pull request by pipo02mix**
A customer has raised this during support. I could not find the values schema to setup a safe margins so I have just made the value configurable

### CODE REVIEW

File: helm/coredns-app/templates/hpa.yaml [hunk 1]
```diff
@@ -11,4 +11,4 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: coredns
-  targetCPUUtilizationPercentage: 70
+  targetCPUUtilizationPercentage: {{ .Values.targetCPUPercentage }}
```
Comment (MarcelMue) [inline helm/coredns-app/templates/hpa.yaml:14]:
```suggestion
  targetCPUUtilizationPercentage: {{ .Values.targetCPUUtilizationPercentage }}
```

File: helm/coredns-app/values.yaml [hunk 2]
```diff
@@ -11,6 +11,7 @@ groupID: 33
 
 minReplicas: 2
 maxReplicas: 50
+targetCPUPercentage: 70
```
Comment (MarcelMue) [inline helm/coredns-app/values.yaml:14]:
```suggestion
targetCPUUtilizationPercentage: 70
```

**Review by MarcelMue — APPROVED** [review]

**Comment by stone-z** [comment]
Optional but maybe it would be a good time to put the HPA options under their own section now that there are multiple. e.g.
```
hpa:
  minReplicas: 2
  maxReplicas: 50
  targetCPUUtilizationPercentage: 70
```
and then
```
  targetCPUUtilizationPercentage: {{ .Values.hpa.targetCPUUtilizationPercentage }}
```

**Comment by pipo02mix** [comment]
@stone-z good point, added

**Review by stone-z — APPROVED** [review]
Cool, thanks!

**Pull request merged**